docs/modules/ROOT/pages/how-to/monitor-http-endpoints.adoc

```diff
@@ -0,0 +1,262 @@
+
+. Verify the configuration
++
+Navigate to the OpenShift Web Console https://console.CLUSTER.cloud/dev-monitoring/ns/<APP_NAMESPACE>/metrics.
++
+Select __Custom Query__ and enter `up{job=~"probe/.*"}`.
+There should be an entry for each target with a value of `1` if the probe was successful.
++
+image::monitoring/blackbox-up.png[List of Blackbox probe jobs https://console.<CLUSTER>.cloud/dev-monitoring/ns/<APP_NAMESPACE>/metrics]
++
+If the probe doesn't show up in the list or the value is `0` see the <<troubleshooting>> section.
+
+. Verify the probe
++
+Select __Custom Query__ and enter `probe_success`.
+There should be an entry for each target with a value of `1` if the probe was successful.
++
+image::monitoring/blackbox-probe-success.png[List of Blackbox probe status https://console.<CLUSTER>.cloud/dev-monitoring/ns/<APP_NAMESPACE>/metrics]
++
+If the probe doesn't show up in the list or the value is `0` see the <<troubleshooting>> section.
+
+== Create an Alert for the Probe
+
+. Create the alert
++
+[source,shell]
+----
+kubectl -n "${APP_NAMESPACE}" apply -f - <<YAML
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: my-app-blackbox-alerts
+spec:
+  groups:
+  - name: my-app-blackbox.rules
+    rules:
+    - alert: EndpointDown
+      expr: probe_success == 0 <1>
+      for: 5m <2>
+      labels:
+        severity: critical
+      annotations:
+        summary: "Endpoint {{ \$labels.instance }} down"
+YAML
+----
+<1> The `probe_success` metric is created by the Blackbox Exporter as an overall status of the probe.
+You can also alert on the individual metrics like `probe_http_status_code` or `probe_duration_seconds`.
+Query all available metrics with `{job=~"probe/.*"}`.
+<2> The `for` clause specifies how long the probe has to fail before the alert is triggered.
+Shorter times might help find problems earlier but might also trigger false positives or flukes.
+
+. Verify the alert was picked up by Prometheus
++
+Navigate to the OpenShift Web Console https://console.<CLUSTER>.cloud/dev-monitoring/ns/<APP_NAMESPACE>/alerts.
++
+image::monitoring/blackbox-alert-list.png[List of Blackbox alerts https://console.<CLUSTER>.cloud/dev-monitoring/ns/<APP_NAMESPACE>/alerts]
++
+The alert should show up in the list.
+If it doesn't show up see the <<troubleshooting>> section.
+
+== Troubleshooting [[troubleshooting]]
+
+=== No Entry is Shown in the List if Querying the `up` Metric `up{job=~"probe/.*"}`
+
+[INFO]
+The probe CRD isn't well-formed or Prometheus hasn't yet scraped the probe.
+
+Prometheus hasn't yet scraped the probe.
+Wait for three more minutes and check again.
+
+If no entry is shown in the list, the `Probe` CRD is most likely is misconfigured.
+Especially check the `spec.prober.url`.
+The format is `HOST:PORT`.
+`http://HOST` isn't valid.
+
+=== The Value of the `up` Metric is `0`
+
+[INFO]
+The probe CRD is well-formed but the blackbox exporter isn't reachable.
```

Comment:
Above, you've capitalized "Blackbox Exporter", but starting here you don't anymore. Please pick one variant and stick to it.